meta-lmp-bsp/conf/machine/include/lmp-machine-custom.inc

```diff
@@ -67,12 +67,20 @@ QB_OPT_APPEND_append_qemuriscv64 = " -bios ${DEPLOY_DIR_IMAGE}/fw_payload.elf"
 ## Freedom U540
 PREFERRED_PROVIDER_virtual/bootloader_freedom-u540 = "u-boot-fio"
 PREFERRED_PROVIDER_u-boot_freedom-u540 = "u-boot-fio"
+PREFERRED_VERSION_u-boot-fio_freedom-u540 ?= "2021.04"
+SPL_BINARY_sota_freedom-u540 = "spl/u-boot-spl.bin"
+UBOOT_SUFFIX_sota_freedom-u540 ?= "itb"
 UBOOT_ENV_freedom-u540 = ""
-IMAGE_BOOT_FILES_freedom-u540 = "fw_payload.bin boot.scr uEnv.txt"
+# TODO: drop this once opensbi moves to 0.9
+RISCV_SBI_PAYLOAD_sota_freedom-u540 = ""
+IMAGE_BOOT_FILES_lmp-base_freedom-u540 = "fw_payload.bin boot.scr uEnv.txt"
```

Comment:
For lmp-base you can just set as IMAGE_BOOT_FILES_freedom-u540, since _sota_ will already cover lmp.

Reply:
Fixed.